### PR TITLE
Pin submodules to branch main

### PR DESCRIPTION
**Pull request by while-false**
To make it easier to pick up changes in submodules I suggest to pin each at main by default.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,8 @@
 [submodule "submodules/eventuals"]
 	path = submodules/eventuals
 	url = https://github.com/3rdparty/eventuals.git
+    branch = main
```
Comment (sp33drac3r) [inline .gitmodules:4]: ...Indentation 
...looks 
weird

**Review by sp33drac3r — APPROVED** [review]

**Review by rjhuijsman — APPROVED** [review]
> Were these review requests intended as "whoever gets to it first" or as "I'd like you all to see this"?

"whoever gets to it first" 

and thanks y'all for your time.

File: .gitmodules (hunk 1 above)
Comment (while-false) [reply .gitmodules:4]: That's so odd, I thought I had fixed this. Thanks though!

**Comment by rjhuijsman** [comment]
> > Were these review requests intended as "whoever gets to it first" or as "I'd like you all to see this"?
> 
> "whoever gets to it first"
> 
> and thanks y'all for your time.

... and did you mean to edit my comment in order to reply to it? That's... Very confusing. 😆

**Pull request merged**